ssd1289/ssd1289.go

```diff
@@ -26,21 +29,20 @@ const height = int16(320)
 
 func New(rs machine.Pin, wr machine.Pin, cs machine.Pin, rst machine.Pin, bus Bus) Device {
 	d := Device{
-		rs:  rs,
-		wr:  wr,
-		cs:  cs,
-		rst: rst,
+		rs:  rs.Set,
+		wr:  wr.Set,
+		cs:  cs.Set,
+		rst: rst.Set,
 		bus: bus,
 	}
-
-	rs.Configure(machine.PinConfig{Mode: machine.PinOutput})
-	wr.Configure(machine.PinConfig{Mode: machine.PinOutput})
-	cs.Configure(machine.PinConfig{Mode: machine.PinOutput})
-	rst.Configure(machine.PinConfig{Mode: machine.PinOutput})
-
-	cs.High()
-	rst.High()
-	wr.High()
+	legacy.ConfigurePinOut(rs)
+	legacy.ConfigurePinOut(wr)
+	legacy.ConfigurePinOut(cs)
+	legacy.ConfigurePinOut(rst)
+
+	cs.Set(true)
+	rst.Set(true)
+	wr.Set(true)
```

Comment:
```suggestion
	cs.High()
	rst.High()
	wr.High()
```

Reply:
Maybe I have overlooked something, but why not add a Set(bool) function in addition to High() and Low() in our new "pin.go"?

If this is working, all changes in the drivers can be reverted, except the used types in New() and structs.

Reply:
> To avoid recursive definition. Set is identical to the type definition and can be easy to misuse. 

That is a valid point. Better to avoid it.

Reply:
> and of course:
> 
> ```
> func (pi PinInput) Get() bool {
> 	return pi()
> }
> ```

Unlike the `Set`, this should not have the recursive implications. Also it brings the `drivers` API just a little closer to the `machine` package API. So seems like a good idea to add it.

Reply:
Yes, lets avoid Toggle. If we want a toggleable API it is easy enough to define with current API:

```go 
type PinToggler struct {
    pin drivers.PinOutput
    current bool
}

func (pt *PinToggler) Toggle() {
    now := !pt.current
    pt.current = now
    pt.pin(now)
}
// and add any other method deemed desirable.
```